cypress/integration/e2e.js

```diff
@@ -0,0 +1,58 @@
+it('Behaves as desired', () => {
+  cy.visit('http://localhost:3000/');
+
+  cy.contains('Title').type('Hello');
+  cy.contains('Description').type('Really long text.');
+  cy.get('button').contains('Add').click();
+
+  cy.get('h3').contains('Hello').should('exist');
+  cy.get('p').contains('Really long text.').should('exist');
+
+  cy.get('#calendar')
```

Comment:
Use data-* attributes to provide context to your selectors and isolate them from CSS or JS changes.